.ci/magician/github/REVIEWER_ASSIGNMENT_COMMENT.md

```diff
@@ -1,4 +1,6 @@
-Hello! I am a robot. Tests will require approval from a repository maintainer to run. Googlers: see go/terraform-auto-test-runs to set up automatic test runs.
+Hello! I am a robot. Tests will require approval from a repository maintainer to run. 
+
+*Googlers*: For automatic test runs see go/terraform-auto-test-runs.
```

Comment:
nit: Maybe use bold instead of italics? And I think the : should be included in the styling.

```suggestion
**Googlers:** For automatic test runs see go/terraform-auto-test-runs.
```